apps/service_acquire/test/acquire/event/handler_test.exs

```diff
@@ -0,0 +1,76 @@
+defmodule Acquire.Event.HandlerTest do
+  use ExUnit.Case
+  import Events, only: [transform_define: 0, load_start: 0, dataset_delete: 0]
+  import Definition, only: [identifier: 1]
+  alias Acquire.ViewState
+
+  @instance Acquire.Application.instance()
+
+  setup do
+    on_exit(fn ->
+      Brook.Test.clear_view_state(@instance, ViewState.Fields.collection())
+      Brook.Test.clear_view_state(@instance, ViewState.Fields.collection())
```

Comment:
is one of these intended to be `ViewState.Destinations.collection()`?

Reply:
Yes, nice catch.